doc/lsst.daf.butler/CHANGES.rst

```diff
@@ -0,0 +1,61 @@
+Butler v22.0 2021-04-01
+=======================
+
+New Features
+------------
+
+* A Butler instance can now be configured with dataId defaults such as an instrument or skymap. [DM-27153]
+* Add ``butler prune-datasets`` command. [DM-26689]
+* Add ``butler query-dimension-records`` command [DM-27344]
+* Add ``--unlink`` option to ``butler prune-collection`` command. [DM-28857]
+* Add progress reporting option for long-lived commands. [DM-28964]
+* Add ``butler associate`` command to add existing datasets to a tagged collection. [DM-26688]
+* Add officially-supported JSON serialization for core Butler classes. [DM-28314]
+* Allow ``butler.get()`` to support dimension record values such as exposure observing day or detector name in the dataID. [DM-27152]
+* Add "direct" ingest mode to allow a file to be ingested retaining the full path to the original file. [DM-27478]
+
+Bug Fixes
+---------
+
+* Fix temporal queries and clarify ``Timespan`` behavior. [DM-27985]
+
+Performance Enhancements
+------------------------
```

Comment:
Can we configure towncrier to drop these empty headings?

Reply:
I wrote these manually. This was me forgetting to remove the placeholders I had put in.